tinygrad/codegen/linearizer.py

```diff
@@ -390,21 +401,20 @@ def linearize(self):
 
     # parse AST
     loaded_buffers:Dict[Union[MemBuffer, ConstBuffer, LocalBuffer], List[UOp]] = {}
-    acc: List[UOp] = []
+    accs: Dict[LazyOp, List[UOp]] = {}
```

Comment:
can you factor out this accs dict refactor to its own PR?

Reply:
yes sir